### PR TITLE
CPU port implementation part one

### PR DESCRIPTION
**Pull request by DanG100**
Background for what will implemented in next PRs:
* The will be grpc client running in VM that is the "CPU port"
* The cpu port receives packets and port information from the dataplane
* The cpu port uses the packet egress ID and the port information to write the frames to the correct places

### CODE REVIEW

File: dataplane/forwarding/fwdport/ports/cpu_test.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func TestCpuWrite(t *testing.T) {
 
 	// Verify that the packet was received and the parsed fields only have
 	// the ETHER_TYPE set to ARP.
-	list := ps.lastPacket.GetParsedFields()
+	list := ps.lastPacket.GetPacket().GetParsedFields()
```
Comment (wenovus) [inline dataplane/forwarding/fwdport/ports/cpu_test.go:109]: is `lastPacket` still an accurate name?

File: dataplane/forwarding/fwdport/ports/genetlink.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	if p.output, err = fwdaction.NewActions(genUpd.Genetlink.GetOutputs(), p.ctx); err != nil {
+		return fmt.Errorf("ports: output actions for port %v failed, err %v", p, err)
+	}
+	return nil
+}
+
+// Write writes a packet out. If successful, the port returns
+// fwdaction.CONSUME.
+func (p *genetlinkPort) Write(fwdpacket.Packet) (fwdaction.State, error) {
+	return fwdaction.CONSUME, nil
```
Comment (wenovus) [inline dataplane/forwarding/fwdport/ports/genetlink.go:83]: Will the grpc server be writing out packets from here?

**Review by wenovus — APPROVED** [review]

File: dataplane/forwarding/fwdport/ports/cpu_test.go (hunk 1 above)
Comment (DanG100) [reply dataplane/forwarding/fwdport/ports/cpu_test.go]: renamed

File: dataplane/forwarding/fwdport/ports/genetlink.go (hunk 2 above)
Comment (DanG100) [reply dataplane/forwarding/fwdport/ports/genetlink.go:83]: no, from the cpu port

**Pull request merged**